packages/x-components/src/components/suggestions/base-suggestions.vue

```diff
@@ -60,14 +60,30 @@
     @Prop()
     protected maxItemsToRender?: number;
 
+    /**
+     * Indicates if the suggestions must be rendered along with its facets.
+     *
+     * @public
+     */
+    @Prop({ default: true })
+    protected showFacets!: boolean;
+
+    /**
+     * When showFacets is true, indicates if the query suggestion without facet must be rendered.
+     *
+     * @public
+     */
+    @Prop({ default: false })
+    protected showQuery!: boolean;
```

Comment:
I will try to find a better name for this.

Reply:
tic tac